packages/Testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Box from '@tds/core-box'
+import Card from '@tds/core-card'
+import Image from '@tds/core-image'
+import Paragraph from '@tds/core-paragraph'
+import safeRest from '@tds/shared-safe-rest'
+
+import styles from './Testimonial.scss'
+
+/**
+ * @version ./package.json
+ */
+/** Testimonial component for displaying testimonial in a standalone, pre-styled component so all testimonials look consistent across the website */
+const Testimonial = ({ testimonial, imgSrc, imgAlt, title, additionalInfo, ...rest }) => {
+  return (
+    <Card {...safeRest(rest)}>
+      <Box between={3}>
+        <Paragraph>&quot;{testimonial}&quot;</Paragraph>
+        <Box inline={3} dangerouslyAddClassName={styles['testimonial-author']}>
```

Comment:
It looks like this line is causing a PropTypes validation error: `<Box inline={3} ...>`

<img width="737" alt="screen shot 2018-09-28 at 3 05 31 pm" src="https://user-images.githubusercontent.com/5464587/46228599-8a3e8600-c330-11e8-8bce-34b41aed2a14.png">